src/test/java/com/appaloosastore/client/AppaloosaClientTest.java

```diff
@@ -458,8 +458,24 @@ public void constructParametersFromUpdateShouldUseDescriptionIfSpecified()
 		params = appaloosaClient.constructParametersFromUpdate(update);
 		assertEquals(3, params.size());
 		assertTrue(params.contains(new BasicNameValuePair("mobile_application_update[description]", update.description)));
-	}	
-	
+	}
+
+	@Test
+	public void constructParametersFromUpdateShouldUseChangelogIfSpecified()
+			throws AppaloosaDeployException, UnsupportedEncodingException, ClientProtocolException, IOException {
+		MobileApplicationUpdate update = new MobileApplicationUpdate();
+		update.id = 1;
+
+		update.changelog = null;
+		List<NameValuePair> params = appaloosaClient.constructParametersFromUpdate(update);
+		assertEquals(2, params.size());
+
+		update.changelog = "";
```

Comment:
Please use a dummy text, so that the test does not appear to check an empty String behavior specifically